internal/cmd/oohelperd/main.go

```diff
@@ -81,9 +81,23 @@ func main() {
 			)
 			return netxlite.NewHTTPClientWithResolver(logger, reso)
 		},
+		NewHTTP3Client: func(logger model.Logger) model.HTTPClient {
+			reso := netxlite.MaybeWrapWithBogonResolver(
+				true, // enabled
+				newResolver(logger),
+			)
+			return netxlite.NewHTTP3ClientWithResolver(logger, reso)
+		},
 		NewDialer: func(logger model.Logger) model.Dialer {
 			return netxlite.NewDialerWithResolver(logger, newResolver(logger))
 		},
+		NewQUICDialer: func(logger model.Logger) model.QUICDialer {
+			return netxlite.NewQUICDialerWithResolver(
```

Comment:
I am wondering whether here we should be using a dialer without a resolver. The existing code seems to feed only endpoints to the dialer anyway. Could you double check my understanding? If I'm right, then I would suggest for us to avoid using a resolver both for QUIC and for TCP a few lines above. We can do this as a cleanup that is included into this pull request.

Reply:
Yes, I agree. I will change this for TCP also.